src/api/java/com/direwolf20/buildinggadgets/api/building/Region.java

```diff
@@ -15,7 +17,18 @@
 /**
  * Represents a region in the world with a finite and nonzero size.
  */
-public final class Region implements IPositionPlacementSequence {
+public final class Region implements IPositionPlacementSequence, Serializable {
```

Comment:
If you want Region to be serilisable, then it needs a serialversion uid...

Reply:
-This reply is for the review message-

When networking, the most efficient way to transfer a bean object is to serialize it into a `byte[]`. Networking of `Region` would most likely happen during the transfer of an `ITemplate`.

Reply:
That is correct, but a Region is so small, that it doesn't make sense to compress it. It's literally just 6 values...

Reply:
True, but it doesn't hurt to have it, right?
Or I can remove it if you firmly believe it won't get used at all.

Reply:
In my opinion it doesn't make sense to optimise streaks, if you leave that in...

Reply:
(basically you'd need streaks>10 for every tile in the save just to balance the effect of saving the blockstate again - the other duplicate variables aren't taken in to account (assuming blockstates need 40 bytes in average - which is quite low if you look at how much space the registry names already takes... It's presumably a lot more))

Reply:
Can you explain where did I saving the blockstate again?